python/revng/internal/cli/_commands/mass_testing/configure.py

```diff
@@ -0,0 +1,57 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import argparse
+import os
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+
+import yaml
+
+from revng.internal.cli.support import run
+
+from ...commands_registry import Command, Options
+
+
+class MassTestingConfigureCommand(Command):
+    def __init__(self):
+        super().__init__(
+            ("mass-testing", "configure"), "Collect binary files and run test-configure"
+        )
+
+    def register_arguments(self, parser: argparse.ArgumentParser):
+        parser.description = "Collect binary files and run test-configure"
+        parser.add_argument("input", help="Input directory")
```

Comment:
This script collects all the file, no filtering takes place, correct?

Reply:
Yes